### DISCUSSION TITLE
How To Remove This error, its annoying in VS Code.

### DISCUSSION THREAD

**Discussion by JenuelDev** [windicss/windicss#444] [Q&A] [answered]
Hi! I need help. I tried changing the settings for Css and scss unkknownRulesAt property to `ignored`. But still, I see this error, it's annoying. 

![image](https://user-images.githubusercontent.com/31676163/131115125-8d6ecfc6-39f4-4e2a-a1ad-750a13d219e9.png)

**Comment by IgnisDa** [comment]
In your `settings.json`

```json
{
  "scss.validate": false,
  "css.validate": false
}
```

Keep in mind that this will supress ALL errors so that might become an issue.

**Reply by JenuelDev** [reply]
oh no, I dont like to suppress every warnings and errors hehehe I just want to ignore the @apply warnings., they make the file color yellow, that is why its very annoying.. but thanks for suggesting..

**Comment by sachinraja** [comment]
You can do this to suppress only the unknownAtRules, you shouldn't have to disable css validation all together:
```json
{
    "scss.lint.unknownAtRules": "ignore"
}
```

**Reply by JenuelDev** [reply]
still not a good answer sorry, I might need this feature in the future for error checking.

**Comment by antfu** [comment, marked as answer]
I would suggest to use `<style lang="postcss"/>`, and it covers all the features you need in your screenshot.

**Reply by JenuelDev** [reply]
oh thanks, already figured it out, but thanks, it has been 3 days of me figuring it out.. but for those who are searching about this, this is the answer.
